cmd/flux/cmd/test.go

```diff
@@ -162,46 +162,45 @@ func (t *Test) Error() error {
 
 // Run the test, saving the error to the err property of the struct.
 func (t *Test) Run(executor TestExecutor) {
-	results, err := executor.Run(t.ast)
-	if err != nil {
-		t.err = err
-		return
-	}
-	defer results.Release()
+	t.err = executor.Run(t.ast, t.execute)
 
+}
+
+func (t *Test) execute(ctx context.Context, results flux.ResultIterator) error {
```

Comment:
Why is this called execute when its supposed to just be reading the result? I found that a bit confusing. Maybe `consume` or just `read` is a better name?

Reply:
I renamed it from `execute` and forgot to change this name.